src/test/java/steps/NewCheckingAccountSteps.java

```diff
@@ -0,0 +1,73 @@
+package steps;
+
+import io.cucumber.java.BeforeAll;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.junit.jupiter.api.Assertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import pages.LoginToBank;
+import utils.Driver;
+
+import java.util.concurrent.TimeUnit;
+
+public class NewCheckingAccountSteps {
+
+    WebDriver driver = Driver.getDriver();
+    private LoginToBank loginToBank = new LoginToBank(driver);
+
+    @BeforeAll
+    public static void setup() {
+
+        WebDriverManager.firefoxdriver().setup();
+    }
+
+    @Given("user is on {string}")
+    public void user_is_on(String string) {
+
+        driver.manage().window().maximize();
+        driver.get("https://dbank-qa.wedevx.co/bank/login");
+        driver.manage().timeouts().implicitlyWait(3, TimeUnit.SECONDS);
+
+
+    }
+    @When("the user enters login {string} and password {string}")
+    public void the_user_enters_login_and_password(String username, String password) throws InterruptedException {
+
+        loginToBank.login(username, password);
+
+    }
+    @Then("the user clicks on {string} and chooses {string} account")
+    public void the_user_clicks_on_and_chooses_account(String string, String string2) throws InterruptedException {
+        Thread.sleep(2000);
```

Comment:
Let's try to not use Thread.sleet but instead use explicit waits

Reply:
Got it!